src/styles/blocks/days.scss

```diff
@@ -0,0 +1,57 @@
+.days {
+  padding: 10px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+
+  &__day {
+    @include square($days-size);
+    border: 1px solid #000;
+    background-color: #eee;
+    transition: $transition;
+
+    &:hover {
+      background-color: pink;
+      transform: translateY(-20px);
+      cursor: pointer;
+    }
+
+    @for $index from 1 through 31 {
+      &--#{$index}::before {
+        content: "#{$index}";
+        height: 100%;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        font-size: 30px;
+      }
+    }
+
+    @function getShift($days-size, $gap, $num) {
+      @return ($days-size + $gap) * $num;
+    }
+
+    $start-day:
+      mon 0,
+      tue getShift($days-size, $gap, 1),
+      wed getShift($days-size, $gap, 2),
+      thu getShift($days-size, $gap, 3),
+      fri getShift($days-size, $gap, 4),
+      sat getShift($days-size, $gap, 5),
+      sun getShift($days-size, $gap, 6);
```

Comment:
Maybe put it in your _variable.scss?
Also, you don't need a function for this, just change how your @each loop works and edit the formula here

Reply:
Fixed